cmd/kube-controller-manager/app/autoscaling.go

```diff
@@ -21,7 +21,12 @@ limitations under the License.
 package app
 
 import (
+	apimeta "k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	discocache "k8s.io/client-go/discovery/cached" // Saturday Night Fever
```

Comment:
if the rest of the this pull earns it, I'll let you have it :)

Reply:
whoops, didn't mean to leave that in :-P